programs/solrand/src/lib.rs

```diff
@@ -226,7 +203,16 @@ pub struct TransferAuthority<'info> {
     pub system_program: Program<'info, System>,
 }
 
-#[error]
+#[derive(Accounts)]
+pub struct Cancel<'info> {
+    #[account(mut)]
```

Comment:
Instead of closing and manually moving lamports around, could you do:
```suggestion
    #[account(mut, close = authority)]
```